src/Raven.Server/Utils/CpuUsage.cs

```diff
@@ -276,7 +311,25 @@ private static unsafe MacInfo GetMacInfo()
             ref FileTime lpKernelTime,
             ref FileTime lpUserTime);
 
-        public static long GetNumberOfActiveCores(Process process)
+        public static double GetNumberOfActiveCores(Process process, ExtensionPointRawData extensionPointRawData)
+        {
+            if (extensionPointRawData == null)
+            {
+                return GetNumberOfActiveCoresInternal(process);
+            }
+            if (extensionPointRawData.ActiveCores >= 0)
+            {
+                return extensionPointRawData.ActiveCores;
+            }
+
+            if (Logger.IsOperationsEnabled)
+            {
+                Logger.Operations($"Received bad data for active cores from extension point : {extensionPointRawData.ActiveCores}");
```

Comment:
This like the admin, what is `from extension point` which one, etc?
Need to be very explicit about this.